### PR TITLE
Awaiting events message

### PR DESCRIPTION
**Pull request by ckj**
This is fairly minor, but it's the first time i've used the translations stuff so I figured I would throw up a PR. Here's what this looks like:

screen shot 2015-12-14 at 12 18 00 pm

/cc @mitsuhiko

### CODE REVIEW

File: src/sentry/static/sentry/app/views/stream.jsx [hunk 1]
```diff
@@ -379,6 +382,22 @@ const Stream = React.createClass({
...
 
+  renderAwaitingEvents() {
+    let org = this.getOrganization();
+    let project = this.getProject();
+
+    return (
+      <div className="box awaiting-events">
+        <div className="wrap">
+          <div className="robot"></div>
+          <h3>Waiting for events…</h3>
```
Comment (mitsuhiko) [inline src/sentry/static/sentry/app/views/stream.jsx]: that one is missing a t()

File: src/sentry/static/sentry/app/views/stream.jsx [hunk 2]
```diff
@@ -379,6 +382,22 @@ const Stream = React.createClass({
...
+          <p>{t('Our error robot is waiting to')} <span className="strikethrough">{t('devour')}</span> {t('recieve your first event.')}</p>
```
Comment (mitsuhiko) [inline src/sentry/static/sentry/app/views/stream.jsx]: this cannot translate. instead do this:

```javascript
          <p>{tct('Our error robot is waiting to [cross:devour] recieve your first event.', {cross: <span className="strikethrough"/>})}</p>
```

Comment (ckj) [inline src/sentry/static/sentry/app/views/stream.jsx]: Thanks - this was the piece I was primarily concerned with. Pushing the change now.

**Comment by ckj** [comment]
Cool, thanks. Gonna merge.

**Pull request merged**